test/e2e/util/mongotester/mongotester_test.go

```diff
@@ -13,7 +13,7 @@ func TestTlsRemoval_RemovesCorrectConfig(t *testing.T) {
 	var opts []*options.ClientOptions
 
 	// configure TLS and hosts
-	opts = withTls(&tls.Config{
+	opts = withTls(&tls.Config{ //nolint
```

Comment:
This was added to appease the linter. I will file an issue to fix this in a separate PR.

Reply:
https://jira.mongodb.org/browse/CLOUDP-80410